setup.py

```diff
@@ -39,75 +40,36 @@ def run_tests(self):
     os.system('twine upload dist/*')
     sys.exit()
 
-packages = ['requests']
-
-requires = [
-    'chardet>=3.0.2,<3.1.0',
-    'idna>=2.5,<2.9',
-    'urllib3>=1.21.1,<1.26,!=1.25.0,!=1.25.1',
-    'certifi>=2017.4.17'
-
-]
-test_requirements = [
-    'pytest-httpbin==0.0.7',
-    'pytest-cov',
-    'pytest-mock',
-    'pytest-xdist',
-    'PySocks>=1.5.6, !=1.5.7',
-    'pytest>=3'
-]
-
-about = {}
-with open(os.path.join(here, 'requests', '__version__.py'), 'r', 'utf-8') as f:
-    exec(f.read(), about)
-
-with open('README.md', 'r', 'utf-8') as f:
-    readme = f.read()
-with open('HISTORY.md', 'r', 'utf-8') as f:
-    history = f.read()
+
+def extract_global_vars(t):
```

Comment:
While you can work out what this doing, it's not immediately obvious. We try to keep the common packaging files in Requests as accessible as possible, so new community members have something simple to reference. We may want a slightly less black magic solution for this.

Reply:
We can move everything into `setup.cfg`. Version can be set from a git tag using `setuptools_scm`.